check_docker

```diff
@@ -396,7 +404,9 @@ def check_cpu(container, warn, crit, _=''):
         max = 100
         evaluate_numeric_thresholds(container=container, value=usage, warn=warn, crit=crit, units='%', name='cpu',
                                     short_name='cpu', min=0, max=max)
-
+    else:
```

Comment:
Rather than repeating the message about the containers not running group up all the checks that are dependent on a running state. This would happen around line 559 and include the checks for memory, cpu, uptime, health, and restarts. 

Also could you add a unit test to make sure the right message is is displayed for checks when the container is stopped. This would be best placed in the 'TestPerform' class.